src/modules/json.wren

```diff
@@ -0,0 +1,245 @@
+import "io" for FileSystem
+
+class JsonOptions {
+    static NIL { 0 }
+    static ESCAPE_SLASHES { 1 }
+    static ABORT_ON_ERROR { 2 }
```

Comment:
I believe that ABORT_ON_ERROR should be a default behaviour, and disabled if necessary.

Reply:
Ok